cpp/msbuild/ice.cpp.props

```diff
@@ -239,39 +228,28 @@
     </Link>
   </ItemDefinitionGroup>
 
-  <ItemDefinitionGroup Condition="'$(IceBuildingUWP)' == 'yes'">
-    <ClCompile>
-      <ProgramDataBaseFileName>$(OutDir)$(TargetName).pdb</ProgramDataBaseFileName>
-    </ClCompile>
-  </ItemDefinitionGroup>
-
   <!--
       Base target names.
   -->
   <Choose>
     <!--
         C++11 mapping target names.
     -->
-    <When Condition="'$(IceCppMapping)' == 'cpp11' and '$(IceBuildingUWP)' != 'yes'">
+    <When Condition="'$(IceCppMapping)' == 'cpp11'">
```

Comment:
Can we do simpler since we use the same TargetName for DynamicLibrary and StaticLibrary?

Reply:
Yes, we can simplify this.